index.js

```diff
@@ -49,5 +50,8 @@ parseString.parseString(xmlStr, (err, result) => {
   result.testsuites.testsuite.forEach(t => {
     console.log(lib.generateTestsuiteSummary(t));
     console.log(lib.generateTestsuiteResult(t));
+    if (program.opts().logs) {
```

Comment:
I think it's better to have opts value returned into a variable in line 26 like:
```
   const options = program.opts();
```
and use it in this like as:
```suggestion
    if (options.logs) {
```

This would prevent needing to call the option parsing function multiple times when new options are needed in the code later.

Reply:
Done :)